pkg/config/config_template.yaml

```diff
@@ -4058,6 +4058,80 @@ api_key:
+  # workers: 10
+
+  ## @param chan_size - integer - optional - default: 5000
+  ## The size of the channel used to send reverse DNS lookups requests to the workers.
+  # chan_size: 5000
+
+  ## @param cache - custom object - optional
+  ## This section configures the cache used by the reverse DNS enrichment component.
+  # cache:
+
+    ## @param enabled - boolean - optional - default: true
+    ## Set to true to enable reverse DNS enrichment caching.
+    #
+    # enabled: true
+
+    ## @param entry_ttl - duration - optional - default: 24h
+    ## The amount of time that a cache entry remains valid before it is expired and removed from the cache.
+    # entry_ttl: 24h
+
+    ## @param clean_interval - duration - optional - default: 2h
+    ## An interval that specifies how often expired entries are removed from the cache to free space.
+    # clean_interval: 2h
+
+    ## @param persist_interval - duration - optional - default: 2h
+    ## An interval that specifies how often the cache is persisted to disk so the cache can be reloaded when the agent is upgraded or restarted.
+    # persist_interval: 2h
+
+    ## @param max_retries - integer - optional - default: 10
+    ## The maximum number of retries to perform when a DNS lookup operation fails, after which the hostname "" is returned and cached for the IP address.
+    # max_retries: 10
+
+    ## @param max_size - integer - optional - default: 1000000
+    ## The maximum size in entries of the cache, above which additional entries will not be cached.
+    #
+    # max_size: 1000000
+
+  ## @param rate_limiter - custom object - optional
+  ## This section configures the rate limiter used by the reverse DNS enrichment component.
+  # rate_limiter:
+
+    ## @param enabled - boolean - optional - default: true
+    ## Set to true to enable the reverse DNS enrichment rate limiter.
+    #
+    # enabled: true
+
+    ## @param limit_per_sec - integer - optional - default: 1000
+    ## The maximum number of reverse DNS lookups allowed per second by the rate limiter.
+    # limit_per_sec: 1000
+
+    ## @param limit_throttled_per_sec - integer - optional - default: 1
+    ## The maximum number of reverse DNS lookups allowed per second when the rate limiter is throttled due to errors exceeding the threshold.
+    # limit_throttled_per_sec: 1
+
+    ## @param throttle_error_threshold - integer - optional - default: 10
+    ## The number of consecutive errors that will trigger the rate limiter to throttle down to limit_throttled_per_sec.
+    # throttle_error_threshold: 10
+
+    ## @param recovery_intervals - integer - optional - default: 5
+    ## The number of intervals over which to throttle the rate limit back to limit_per_sec after being throttled.
+    # recovery_intervals: 5
+
+    ## @param recovery_interval - duration - optional - default: 5s
+    ## The interval between incrementally increasing the rate limit back to limit_per_sec after being throttled.
+    # recovery_interval: 5s
```

Comment:
I can't find those four configs declared or used anywhere in the agent repo ?

Reply:
these are related to this PR that will be merged before this one!: https://github.com/DataDog/datadog-agent/pull/27876